components/GlobalLink/GlobalLink.styled.js

```diff
@@ -0,0 +1,58 @@
+import Link from "next/link";
+import styled from "styled-components";
+
+export const LinkButton = styled(Link)`
```

Comment:
Please rename it to StyledLink, this name makes it unclear if the component is a button or a Link